scripts/runtime-code-shasum.sh

```diff
@@ -28,6 +35,7 @@ ${TAR} -c --sort=name --owner=root:0 --group=root:0 --mode 644 --mtime='UTC 2020
     joystream-node-armv7.Dockerfile \
     node \
     $(test -n "$TEST_NODE" && echo "$TEST_PROPOSALS_PARAMETERS_PATH") \
+    $PROPOSALS_PARAMETERS_FILE \
```

Comment:
> Wondering wethere the built docker image should also store the proposals parameters.json file which ends up being used for the build as well.
Since we are storing ALL_PROPOSALS_PARAMETERS_JSON as a file, it will be picked up by the command `COPY . /joystream` of our Dockerfile